### ISSUE TITLE
Add TypeScript module declaration

### ISSUE THREAD

**Issue by lorenzleutgeb** [log4js-node/log4js-api#1]
Using this package I run into the following error:

```
error TS7016: Could not find a declaration file for module '@log4js-node/log4js-api'.
              '.../node_modules/@log4js-node/log4js-api/lib/index.js' implicitly has
              an 'any' type.
Try `npm install @types/log4js-node__log4js-api` if it exists or add a new
declaration (.d.ts) file containing `declare module 'log4js-node__log4js-api';`
```

For now I have solved this by providing a module declaration myself which just exports `getLogger(): any`. I don't know what would be the best solution to the problem since I am rather new to TypeScript.

It would be nice if you could provide module declarations, like you already do for `log4js`. Thank you.

**Comment by ronnyroeller** [comment]
+1

**Comment by nomiddlename** [comment]
A fix for this has been published to npm in version 1.0.1.
